skyrl-train/skyrl_train/generators/skyrl_gym_generator.py

```diff
@@ -58,9 +58,10 @@ def __init__(
         self.max_turns = generator_cfg.max_turns
         self.batched = generator_cfg.batched
         self.use_conversation_multi_turn = generator_cfg.use_conversation_multi_turn
+        self.chat_template = generator_cfg.chat_template
 
```

Comment:
```suggestion
self.custom_chat_template = get_custom_chat_template(generator_cfg.chat_template)
```

Having a `self.chat_template` might be confusing. let merge these two lines